presto-hive/src/test/java/com/facebook/presto/hive/benchmark/TpcdsTableName.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.hive.benchmark;
```

Comment:
This class is better to be put in `com.facebook.presto.tests.tpcds` package, in analogous to the `com.facebook.presto.tests.tpch` package. Package `com.facebook.presto.hive.benchmark` is for benchmarks and tests themselves.